lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2271,6 +2271,10 @@ def add_collection3d(self, col, zs=0, zdir='z'):
         collection = super().add_collection(col)
         return collection
 
+    @_preprocess_data(replace_names=["xs", "ys", "zs", "s",
+                                     "edgecolors", "c", "facecolor",
+                                     "facecolors", "color"],
+                      label_namer=None)
```

Comment:
```suggestion
                                     "facecolors", "color"])
```
It's corrct that there is no canonical label namer in this case, but `None` is the default so can be left out.